File: backend2/DataAccess/DaoGenerated/システム共有/tt_afuser_sisyoDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             ユーザー所属部署（支所）テーブル
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afuser_sisyoDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afuser_sisyo";
		public string userid { get; set; }                        //ユーザーID
		public string sisyocd { get; set; }                       //部署（支所）コード
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afuser_sisyoDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             ユーザー所属部署（支所）テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afuser_sisyoDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afuser_sisyo"
        Public Property userid As String                        'ユーザーID
        Public Property sisyocd As String                       '部署（支所）コード
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
